src/tanks/Tank.js

```diff
@@ -54,6 +53,7 @@ import {
 } from '../casus/userInteraction/CasusSpecialVariables.js';
 
 import type Battleground from '../battleground/Battleground.js';
+import ContainerBlock from '../casus/blocks/ContainerBlock.js';
```

Comment:
This should be in the real imports section, not in the type imports section. (type imports get stripped out by the code we send to clients, so they don't actually cause JS cycles, that is why they are in a different section)